### PR TITLE
Uifixes

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9zyr6abgvf2t6g4503vev8dk6g

[download apk](http://filestorage.bluewallet.io:1488/1579491656.apk)

**Comment by ncoelho** [comment]
lgtm

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/01xzf43bwu9f9nq81bh691vz34

[download apk](http://filestorage.bluewallet.io:1488/1579565131.apk)

File: screen/wallets/list.js [hunk 1]
```diff
@@ -68,7 +67,7 @@ export default class WalletsList extends Component {
         console.log('fetch all wallet txs took', (end - start) / 1000, 'sec');
       } catch (error) {
         noErr = false;
-        alert(error);
+        console.log(error);
```
Comment (Overtorment) [inline screen/wallets/list.js]: does it mean that electrum errors wont throw alerts anymore? it will be silent?

File: screen/wallets/list.js [hunk 2]
```diff
@@ -262,7 +260,6 @@ export default class WalletsList extends Component {
       }
     } catch (Err) {
       noErr = false;
-      alert(Err);
```
Comment (Overtorment) [inline screen/wallets/list.js]: why did you decide to remove alerts here? whats the reasoning here?

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/t360yypm23ytfeyxe54cgcx99c

[download apk](http://filestorage.bluewallet.io:1488/1579721685.apk)

File: screen/wallets/list.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/list.js]: The electrum errors are separate from this.

File: screen/wallets/list.js (hunk 2 above)
Comment (marcosrdz) [reply screen/wallets/list.js]: this was recently put in place. but there were a lot of "undefined" errors popping up causing user annoyance

**Pull request merged**